### PR TITLE
Marcandea validation ean

### PR DESCRIPTION
**Pull request by Marcandea**
This is Two Tests for verify stringEAN. (Completed)

### CODE REVIEW

File: src/main/java/org/fundacionjala/coding/marcoslara/spinwords/SpinWords.java [hunk 1]
```diff
@@ -1,32 +0,0 @@
-package org.fundacionjala.coding.marcoslara.spinwords;
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/marcoslara/spinwords/SpinWords.java:1]: Why did you remove this class?
please keep it.

File: src/test/java/org/fundacionjala/coding/marcoslara/spinwords/SpinWordsTest.java [hunk 2]
```diff
@@ -1,65 +0,0 @@
-package org.fundacionjala.coding.marcoslara.spinwords;
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/coding/marcoslara/spinwords/SpinWordsTest.java:1]: Same as above

File: src/test/java/org/fundacionjala/coding/Marcan/TestValidationEAN.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        ValidationEAN stringEAN = new ValidationEAN();
+        assertTrue(stringEAN.verifyStringEAN("4003301018398"));
+    }
+
+    /**
+     * This is a Test for verify if is false the result (equal to ultimate number).
+     */
+    @Test
+    public void secondTestForAssertFalseOfString() {
+        ValidationEAN stringEAN = new ValidationEAN();
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/coding/Marcan/TestValidationEAN.java]: You are repeating this code in all your tests
Please put them in @before method or convert it to a static method.

File: src/main/java/org/fundacionjala/coding/marcoslara/spinwords/SpinWords.java (hunk 1 above)
Comment (Marcandea) [reply src/main/java/org/fundacionjala/coding/marcoslara/spinwords/SpinWords.java:1]: Sorry, I thought it did not affect, it appears when I pull

File: src/test/java/org/fundacionjala/coding/Marcan/TestValidationEAN.java (hunk 3 above)
Comment (Marcandea) [reply src/test/java/org/fundacionjala/coding/Marcan/TestValidationEAN.java]: Ok, thanks, I will implement.

File: src/main/java/org/fundacionjala/coding/marcoslara/spinwords/SpinWords.java (hunk 1 above)
Comment (carledriss) [reply src/main/java/org/fundacionjala/coding/marcoslara/spinwords/SpinWords.java:1]: Please apply the workaround
1. git reset HEAD~3 (to revert the commits)
2. git commit again without removing those files
3. git push --force

Comment (Marcandea) [reply src/main/java/org/fundacionjala/coding/marcoslara/spinwords/SpinWords.java:1]: I do this with check out in the Validation EAN project.

Comment (Marcandea) [reply src/main/java/org/fundacionjala/coding/marcoslara/spinwords/SpinWords.java:1]: ?

**Comment by carledriss** [comment]
@Marcandea 
Please ask for help to your teammates
